cluster-autoscaler/main.go

```diff
@@ -175,13 +175,13 @@ func createKubeClient() kube_client.Interface {
 func registerSignalHandlers(autoscaler core.Autoscaler) {
 	sigs := make(chan os.Signal, 1)
 	signal.Notify(sigs, os.Interrupt, os.Kill, syscall.SIGTERM, syscall.SIGQUIT)
-	glog.Info("Registered cleanup signal handler")
+	glog.V(1).Info("Registered cleanup signal handler")
 
 	go func() {
 		<-sigs
-		glog.Info("Received signal, attempting cleanup")
+		glog.V(1).Info("Received signal, attempting cleanup")
 		autoscaler.ExitCleanUp()
-		glog.Info("Cleaned up, exiting...")
+		glog.V(1).Info("Cleaned up, exiting...")
```

Comment:
I would make this a V(0). CA restart is a pretty important event.

Reply:
I don't want to have anything on V(0) so that it is possible to turn off logs completely. V(1) should be ok as the default level is 4.

Reply:
Fair enough. Though we do have some V(0) logs now, which I don't think are more important to log than CA restart:
```
> $ grep -nr "V(0)" | grep -v vendor | wc -l
9
```

Reply:
Ok, next PR :).